src/arithmetic/arithmetic_expression.c

```diff
@@ -551,11 +577,22 @@ bool AR_EXP_ContainsFunc(const AR_ExpNode *root, const char *func) {
 	return false;
 }
 
-bool inline  AR_EXP_IsConstant(const AR_ExpNode *exp) {
+AR_ExpNode *AR_EXP_SeekFunc(AR_ExpNode *root, const char *func) {
```

Comment:
We should note that this function returns the first occurrence of `func` it finds, there might be additional ones.